rtengine/impulse_denoise.h

```diff
@@ -188,10 +180,6 @@ SSEFUNCTION void ImProcFunctions::impulse_nr (LabImage* lab, double thresh)
 
                 for (i1 = max(0, i - 2); i1 <= min(i + 2, height - 1); i1++ )
                     for (j1 = 0; j1 <= j + 2; j1++ ) {
-                        if (i1 == i && j1 == j) {
```

Comment:
That was already covered by the following:

if (impish[i1][j1]) {
                            continue;
                        }